.travis.yml

```diff
@@ -11,8 +11,8 @@ rvm:
 
 before_script:
   - bundle exec rake db:create
+  - bundle exec rake db:migrate
```

Comment:
Instead of running the migrations, let's setup from `schema.rb`: `bundle exec rails db:setup`. The setup here will edit the schema before running the specs.